### PR TITLE
Add endpoint to publish polls

### LINKED ISSUE #210: Confirm we can build all leader content types (questions)
# Overview
Currently, there only seems we can build a subset of the expected leader questions base on reviewing the form validation: https://github.com/PowerlineApp/powerline-server/blob/develop/backend/src/Civix/ApiBundle/Form/Type/Poll/QuestionType.php

Current questions: 
* event (group_event)
* payment request / crowd fundraiser (group_payment_request)
* petition (group_petition)

Missing questions:
* poll
* discussion (group_news)

# Action / Work
1. If we are missing the above questions, let's build those out just like the rest of the questions
2. If we are not missing those questions, please document here where that information is.

### PR DESCRIPTION
**Pull request by igaponov**
Closes #210

### CODE REVIEW

File: backend/src/Civix/ApiBundle/EventListener/LeaderContentSubscriber.php [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+use Civix\CoreBundle\Event\Poll\QuestionEvent;
+use Civix\CoreBundle\Event\PollEvents;
 use Civix\CoreBundle\Service\Micropetitions\PetitionManager;
 use Civix\CoreBundle\Service\Poll\CommentManager;
 use Civix\CoreBundle\Service\Settings;
 use Doctrine\ORM\EntityManager;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
-class PetitionSubscriber implements EventSubscriberInterface
+class LeaderContentSubscriber implements EventSubscriberInterface
```
Comment (austinpapp) [inline backend/src/Civix/ApiBundle/EventListener/LeaderContentSubscriber.php]: this event listener is handling everything from user micropetitions to leader questions?

File: backend/src/Civix/CoreBundle/Validator/Constraints/PublishedPollAmount.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+namespace Civix\CoreBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class PublishedPollAmount extends Constraint
+{
+    public $message = 'Published poll amount has reached';
```
Comment (austinpapp) [inline backend/src/Civix/CoreBundle/Validator/Constraints/PublishedPollAmount.php]: 'Published poll amount has _been_ reached'

**Comment by austinpapp** [comment]
@igaponov i see this publishes all of the leader generated content. however, i'm not sure we confirmed that we have the `poll` and `discussion` logic available to be created.

File: backend/src/Civix/ApiBundle/EventListener/LeaderContentSubscriber.php (hunk 1 above)
Comment (igaponov) [inline backend/src/Civix/ApiBundle/EventListener/LeaderContentSubscriber.php]: It handles both micropetitions and questions (as they have similar functionalities).

File: backend/src/Civix/CoreBundle/Validator/Constraints/PublishedPollAmount.php (hunk 2 above)
Comment (igaponov) [inline backend/src/Civix/CoreBundle/Validator/Constraints/PublishedPollAmount.php]: Done

**Comment by igaponov** [comment]
@austinpapp simple poll = type `group` for group questions and `representative` for representatives.
There is no any entity similar to `discussion`, what entity do you mean?

**Pull request merged**